_includes/codeTable.html

```diff
@@ -247,7 +248,7 @@ <h2 class="modal-title">{{ release.name[page.lang] }}</h2>
         {%- if release.status != nil -%}
           <p>{{ site.data.i18n.tables.status[page.lang] }}: {{ release.status }}</p>
```

Comment:
Should all the field descriptions be Strong?

Reply:
Yes, I missed 2 fields.